Common/cpp/NativeModules/NativeReanimatedModule.cpp

```diff
@@ -321,6 +321,13 @@ jsi::Value NativeReanimatedModule::configureLayoutAnimation(
   return jsi::Value::undefined();
 }
 
+jsi::Value NativeReanimatedModule::disableExiting(
```

Comment:
Will this work?
```suggestion
void NativeReanimatedModule::disableExiting(
```

Reply:
It will. From what I've seen there are 2 approaches. The `methodMap` takes only functions that return `jsi::Value`. So we can either return `jsi::Value::undefined()` here or make this function return void and return `jsi::Value::undefined()` in the wrapper that is inserted into the map in `NativeReanimatedModuleSpec`.

Reply:
Let's ask @piaskowyk tomorrow.